torch/ao/quantization/backend_config/onednn.py

```diff
@@ -0,0 +1,189 @@
+# linear leaky_relu, linear module + leaky_relu module
+linear_configs.append(
+    BackendPatternConfig((nn.LeakyReLU, nn.Linear))
+        .set_dtype_configs(linear_dtype_configs)  # noqa: E131
+        .set_fuser_method(reverse_sequential_wrapper2(nni.LinearLeakyReLU))
+        .set_fused_module(nni.LinearLeakyReLU))
+# linear leaky_relu, linear module + functional leaky_relu
+linear_configs.append(
+    BackendPatternConfig((F.leaky_relu, nn.Linear))
+        .set_dtype_configs(linear_dtype_configs)  # noqa: E131
+        .set_fuser_method(reverse_sequential_wrapper2(nni.LinearLeakyReLU))
+        .set_fused_module(nni.LinearLeakyReLU))
+# linear leaky_relu, linear module + BN + leaky_relu
+linear_configs.append(
+    BackendPatternConfig((nn.LeakyReLU, (nn.BatchNorm1d, nn.Linear)))
+        .set_dtype_configs(linear_dtype_configs)  # noqa: E131
+        .set_fuser_method(reverse3(_fuse_linear_bn_leaky_relu))
+        .set_fused_module(nni.LinearLeakyReLU))
+
+# 1.2 linear module + leaky_relu, fused module configs
+# linear leaky_relu, fused module
+linear_configs.append(
+    BackendPatternConfig(nni.LinearLeakyReLU)
+        .set_observation_type(observation_type)  # noqa: E131
+        .set_dtype_configs(linear_dtype_configs)
+        .set_root_module(nn.Linear)
+        .set_reference_quantized_module(nnqr.Linear))
+# 1.3 functional linear + leaky_relu configs
+# linear leaky_relu, functional linear + leaky_relu module
+linear_configs.append(
+    BackendPatternConfig((nn.LeakyReLU, F.linear))
+        .set_observation_type(observation_type)  # noqa: E131
+        .set_dtype_configs(linear_dtype_configs))
+# linear leaky_relu, functional linear + functional leaky_relu
+linear_configs.append(
+    BackendPatternConfig((F.leaky_relu, F.linear))
+        .set_observation_type(observation_type)  # noqa: E131
+        .set_dtype_configs(linear_dtype_configs))
+
+# =====================
+# |  BACKEND CONFIGS  |
+# =====================
+
+def get_onednn_backend_config() -> BackendConfig:
+    """
+    Return the `BackendConfig` for PyTorch's native ONEDNN backend.
+    """
+    return BackendConfig("onednn") \
+        .set_backend_pattern_configs(conv_configs) \
+        .set_backend_pattern_configs(linear_configs) \
+        .set_backend_pattern_configs(_get_binary_op_configs(binary_op_dtype_configs)) \
+        .set_backend_pattern_config(_get_cat_config(default_op_dtype_configs)) \
+        .set_backend_pattern_configs(_get_default_op_configs(default_op_dtype_configs)) \
+        .set_backend_pattern_configs(_get_fixed_qparams_op_configs(fixed_qparams_op_dtype_configs)) \
+        .set_backend_pattern_configs(_get_share_qparams_op_configs(share_qparams_op_dtype_configs)) \
+        .set_backend_pattern_configs(_get_bn_configs(default_op_dtype_configs)) \
+        .set_backend_pattern_configs(_get_rnn_op_configs(rnn_op_dtype_configs)) \
+        .set_backend_pattern_configs(_get_embedding_op_configs(embedding_op_dtype_configs))
```

Comment:
are you sure onednn actually supports all the ops here?

Reply:
Well, I thought we needed all these configs set so I just copied them here. I will try to remove them.

Reply:
They are removed